### DISCUSSION TITLE
Scrollable container bindings do not work in modal screen?

### DISCUSSION THREAD

**Discussion by pawamoy** [Textualize/textual#3629] [Q&A]
I'm pushing a modal screen which is composed a static widget inside of a scrollable container. Upon showing the modal, the container does have a vertical scrollbar, but does not answer to its [documented bindings](https://textual.textualize.io/api/containers/#textual.containers.ScrollableContainer.BINDINGS), i.e. arrow up/down to scroll. The mouse wheel does scroll though. Is this expected?

```python
from textual.app import App
from textual.binding import Binding
from textual.containers import ScrollableContainer
from textual.screen import ModalScreen
from textual.widgets import Static, Footer


class Modal(ModalScreen):
    def compose(self):
        yield ScrollableContainer(Static(f"{'MODAL ' * 8}\n" * 128), id="modal")


class MyApp(App):
    BINDINGS = [
        Binding("m", "modal", "Show modal"),
    ]

    def compose(self):
        yield Footer()

    def action_modal(self):
        self.push_screen(Modal())


app = MyApp()
app.run()
```

**Comment by TomJGooding** [comment]
I think the problem is that `ScrollableContainer` doesn't receive focus, so not related to the modal screen. 

I've raised #3632.

**Reply by pawamoy** [reply]
Ah yes, that would make sense. Thanks a lot!
